src/pages/comparison/comparison.tsx

```diff
@@ -27,6 +27,27 @@ import {
   BaseMap,
 } from "../../types";
 import { LayersPanel } from "../../components/layers-panel/layers-panel";
+import { EXAMPLES } from "../../constants/i3s-examples";
+
+import DarkMap from "../../../public/icons/dark-map.png";
+import LightMap from "../../../public/icons/light-map.png";
+import TerrainMap from "../../../public/icons/terrain-map.png";
+
+export const BASE_MAPS: BaseMap[] = [
```

Comment:
FYI: It is copy-pasted from constant because of failing unit tests. There was an error with processing "png" files in the test environment.

Reply:
Yes, I saw it, when i tried to move it back to the constants.